### ISSUE TITLE
[ Feature Request #2837 ] Easy To Use Notation Input

### ISSUE THREAD

**Issue by matevz** [canorusmusic/canorus#64]
**Reported by suamor on 2011-10-31 10:57 UTC**
Date: 2006-Nov-03 22:45
Sender: tubasoldier
Logged In: YES 
user\_id=33209
Browser: Mozilla/5.0 \(X11; U; Linux i686; en-US; rv:1.8.0.7\) Gecko/20060921
Ubuntu/dapper-security Firefox/1.5.0.7

I like the path that Canorus has laid out. As a result
I would much rather support this project than
Rosegarden. \(as it wants to be a sequencer before a
notation editor.\) 

I liked NoteEdit but I always thought it took way to
long to input notes onto the staff. RoseGarden seems to
be a little better but it is still really awkward to
input notes. I always thought Finale was quite awkward
in this as well. A small windows program called
NoteWorthy Composer seems to have come up with a great
way for note input. It treats the music staff more like
a text editor. Here is a few of the basics:

\- A blinking cursor that moves horizontally and
vertically along the staff. This cursor is moved with
the arrow keys.
\- &lt;Enter&gt; will input a note
\- &lt;Space&gt; will input a rest
\- &lt;ctl + Enter&gt; will input a chord member
\- &lt;backspace&gt; will delete the note before the cursor
\- &lt;Del&gt; will delete the note after the cursor.

Basically there is a computer keyboard shortcut for all
notation input. This makes it extremely easy to input
music without a midi keyboard. Point and Click with the
mouse to input notes is very tedious. 

Please consider making keyboard shortcuts for simple
note entry. However, keeping the functionality of input
or editing with the mouse would also be helpful.

**Comment by matevz** [comment]
**Updated by suamor on 2011-10-31 12:21 UTC**
- **assigned_to**: nobody --> suamor

**Comment by matevz** [comment]
**Commented by suamor on 2018-06-07 09:09 UTC**
Some questions on it (to discuss):
1) How do we activate cursor-based editing
2) Does Insert Mode already activates cursor-based editing
3) Where is the cursor placed when we start editing
